network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -212,33 +218,36 @@ <h2 class="h3-heading">{% trans "Security" %}</h2>
         </section>
 
+        {% if product.product_type == 'general' %}
+          <section class="criterion-group">
+            <h2 class="h3-heading">{% trans "Artificial Intelligence" %}</h2>
+            {% with product.uses_ai|extended_yes_no as uses_ai %}
+            {% trans "Does the product use AI?" as uses_ai_label %}
+            {% include "fragments/product_criterion.html"  with  label=uses_ai_label  value=uses_ai   %}
 
-        <section class="criterion-group">
-          <h2 class="h3-heading">{% trans "Artificial Intelligence" %}</h2>
-          {% with product.uses_ai|extended_yes_no as uses_ai %}
-          {% trans "Does the product use AI?" as uses_ai_label %}
-          {% include "fragments/product_criterion.html"  with  label=uses_ai_label  value=uses_ai   %}
-
-          {% if uses_ai != "No" %}
-            {% trans "Does the AI use your personal data to make decisions about you?" as ai_uses_personal_data %}
-            {% include "fragments/product_criterion.html"  with  label=ai_uses_personal_data  value=product.ai_uses_personal_data|extended_yes_no   %}
+            {% if uses_ai != "No" %}
+              {% trans "Does the AI use your personal data to make decisions about you?" as ai_uses_personal_data %}
+              {% include "fragments/product_criterion.html"  with  label=ai_uses_personal_data  value=product.ai_uses_personal_data|extended_yes_no   %}
 
-            {% trans "Does the company allow users to see how the AI works?" as ai_is_transparent %}
-            {% include "fragments/product_criterion.html"  with  label=ai_is_transparent  value=product.ai_is_transparent|extended_yes_no           %}
+              {% trans "Does the company allow users to see how the AI works?" as ai_is_transparent %}
+              {% include "fragments/product_criterion.html"  with  label=ai_is_transparent  value=product.ai_is_transparent|extended_yes_no           %}
 
-            <p class="pni-product-smaller-body mb-0">{{product.ai_helptext}}</p>
-          {% endif %}
-          {% endwith %}
-        </section>
+              <p class="pni-product-smaller-body mb-0">{{product.ai_helptext}}</p>
+            {% endif %}
+            {% endwith %}
+          </section>
+        {% endif %}
 
 
-        <section class="criterion-group">
-          <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
+        {% if product.product_type == 'general' %}
+          <section class="criterion-group">
+            <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
 
-          <div>
-            {% include "fragments/product_contact.html" with product=product %}
-          </div>
-        </section>
+            <div>
+              {% include "fragments/product_contact.html" with product=product %}
+            </div>
+          </section>
+        {% endif %}
```

Comment:
(for some reason GitHub doesn't let me provide code suggestion)

According to the spreadsheet (line 77-80), contact info fields should be for all products.

Reply:
Thread in slack, but the summary is "the _fields_ exist in the base class, but we should only _render_ it for general products right now" so that's why it's in an `if` block here.

Reply:
Ah ok! So confusing! I was using the sheet as reference and thought they were for-all fields.